node/src/chain_spec/snow.rs

```diff
@@ -1,6 +1,6 @@
 use ice_runtime::{
 	AccountId, AuraConfig, BalancesConfig, CouncilConfig, EVMConfig, EthereumConfig, GenesisConfig, GrandpaConfig,
-	Signature, SudoConfig, SystemConfig, WASM_BINARY, currency::ICY
+	Signature, SudoConfig, SystemConfig,AirdropConfig, WASM_BINARY, currency::ICY
```

Comment:
Should we just do this for snow.rs or also for others like ice.rs, frost.rs. Whatever envs we have at the moment in web3labs's repo main branch